### PR TITLE
LSIF: Add relevant jobs to repo settings code intel page

### PR DESCRIPTION
**Pull request by efritz**
This PR adds a panel to the repo settings code intelligence page that shows the most recent active/queued jobs for a repository as well as some recent failures.

Screen Shot 2019-11-08 at 8 48 39 AM

Screen Shot 2019-11-08 at 9 03 28 AM

Screen Shot 2019-11-08 at 8 48 56 AM

Screen Shot 2019-11-08 at 8 48 32 AM

### CODE REVIEW

File: web/src/repo/settings/RepoSettingsCodeIntelligencePage.scss [hunk 1]
```diff
@@ -1,8 +1,24 @@
 .repo-settings-code-intelligence-page {
     width: 100%;
+
+    &-collapsible {
+        border: 1px solid #2b3750;
```
Comment (attfarhan) [inline web/src/repo/settings/RepoSettingsCodeIntelligencePage.scss]:
```suggestion
        border: 1px solid $color-border;
```

nit: you can use `$color-border` https://sourcegraph.com/github.com/sourcegraph/sourcegraph@d5bb53fb4db4ecb9b1b81724df314983ad6da1ee/-/blob/shared/src/global-styles/colors.scss#L43:17

File: web/src/repo/settings/RepoSettingsCodeIntelligencePage.tsx [hunk 2]
```diff
@@ -47,6 +82,42 @@ export const RepoSettingsCodeIntelligencePage: FunctionComponent<Props> = ({ rep
...
     )
 
+    const fetchJobs = (name: string, state: GQL.LSIFJobState): Observable<GQL.ILSIFJobConnection> =>
+        fetchLsifJobs({
+            query: `convert ${name}`,
+            state,
+            first: 5,
+        })
+
+    const activeLsifJobs = useObservable(useMemo(() => fetchJobs(repo.name, GQL.LSIFJobState.PROCESSING), [repo.name]))
```
Comment (attfarhan) [inline web/src/repo/settings/RepoSettingsCodeIntelligencePage.tsx]: Do errors need to be caught for these?

Would expect the observable to be piped to `catchError` like in https://github.com/sourcegraph/sourcegraph/pull/6509/files#diff-a2fff41eb6fe2bd941e30d47a656e459R43-R45. 

 cc @felixfbecker

Comment (efritz) [reply web/src/repo/settings/RepoSettingsCodeIntelligencePage.tsx]: I've updated it to catch/display errors.

Comment (felixfbecker) [reply web/src/repo/settings/RepoSettingsCodeIntelligencePage.tsx]: @attfarhan no, `useObservable` handles errors: https://github.com/sourcegraph/sourcegraph/blob/bed16f7fe79adc6c8e1fe2c697ce8d64f542b6b5/web/src/util/useObservable.ts#L32

The error would be caught by the nearest `ErrorBoundary` (which would be the page I think). The best way for you to verify here @efritz is to simply throw an exception in `fetchJobs` and see if the error view looks good.

Comment (efritz) [reply web/src/repo/settings/RepoSettingsCodeIntelligencePage.tsx]: Confirmed to work, looks good. Simplified error checking.

**Pull request merged**